mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -202,7 +202,7 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 	METHOD method_23325 getParticleZ (D)D
 		ARG 1 widthScale
 	METHOD method_23326 getVelocityMultiplier ()F
-	METHOD method_23327 setPos (DDD)V
+	METHOD method_23327 setPosition (DDD)V
```

Comment:
pos imo is fine (shorter)

Reply:
Do I rename `Position` to `Pos` in all methods?

Reply:
Hmm, good question. This method compared to others is fairly frequently used as all xyz direct modifications has been migrated to this method in 1.15 snapshots.